Cargo.toml

```diff
@@ -7,6 +7,22 @@ description = "Transit data management"
 repository = "https://github.com/CanalTP/transit_model"
 keywords = ["ntfs", "gtfs", "netex", "navitia", "transit"]
 edition = "2018"
+exclude = [
+	".gitignore",
+	".mergify.yml",
+	".travis.yml",
+	"Cargo.lock",
```

Comment:
this file appears in the package

Reply:
Indeed, I didn't saw that. I guess I can just remove them all... This PR is gonna be simpler :smile:

Reply:
I've removed all the occurrences of `Cargo.lock`.